doc/mkdocs/docs/api/basic_json/erase.md

```diff
@@ -103,14 +119,16 @@ Strong exception safety: if an exception occurs, the original value stays intact
        - strings and binary: linear in the length of the member
        - other types: constant
 3. `log(size()) + count(key)`
-4. Linear in distance between `idx` and the end of the container.
+4. See 3.
```

Comment:
Nitpicky, but I'd rather repeat `Logarithmic in the size of the container.` here. The level of indirection for that short sentence confuses me.